sklearn/linear_model/tests/test_coordinate_descent.py

```diff
@@ -712,3 +713,18 @@ def test_enet_float_precision():
             assert_array_almost_equal(intercept[np.float32],
                                       intercept[np.float64],
                                       decimal=4)
+
+
+def test_enet_l1_ratio():
+    # Test that an error message is raised if an estimator that
+    # uses _alpha_grid is called with l1_ratio=0
+    msg = ("Automatic alpha grid generation is not supported for l1_ratio=0. "
+           "Please supply a grid by providing your estimator with the "
+           "appropriate `alphas=` argument.")
+    X = np.array([[1, 2, 4, 5, 8], [3, 5, 7, 7, 8]]).T
+    y = np.array([12, 10, 11, 21, 5])
+
+    assert_raise_message(ValueError, msg,
+                         ElasticNetCV(l1_ratio=0).fit, X, y)
+    assert_raise_message(ValueError, msg,
+                         MultiTaskElasticNetCV(l1_ratio=0).fit, X, X)
```

Comment:
I am sorry for a possible stupid question, but this commit broke an unrelated test (Logistic regression) on AppVeyor:
```
======================================================================
FAIL: sklearn.linear_model.tests.test_logistic.test_logistic_regression_sample_weights
----------------------------------------------------------------------
Traceback (most recent call last):
  File "C:\Python27\lib\site-packages\nose\case.py", line 197, in runTest
    self.test(*self.arg)
  File "C:\Python27\lib\site-packages\sklearn\linear_model\tests\test_logistic.py", line 638, in test_logistic_regression_sample_weights
    assert_array_almost_equal(clf_cw.coef_, clf_sw.coef_, decimal=4)
  File "C:\Python27\lib\site-packages\numpy\testing\utils.py", line 842, in assert_array_almost_equal
    precision=decimal)
  File "C:\Python27\lib\site-packages\numpy\testing\utils.py", line 665, in assert_array_compare
    raise AssertionError(msg)
AssertionError: 
Arrays are not almost equal to 4 decimals
(mismatch 20.0%)
 x: array([[ 2.5404,  0.    ,  0.    , -0.3094, -0.4925]])
 y: array([[ 2.5405,  0.    ,  0.    , -0.3094, -0.4924]])
sklearn.decomposition.tests.test_dict_learning.test_dict_learning_reconstruction_parallel: 4.1117s
```
but the same tests pass on my computer and on Travis, how do I proceed here?